go/indexer/services/l2/bridge/standard_bridge.go

```diff
@@ -26,9 +26,8 @@ func (s *StandardBridge) GetWithdrawalsByBlockRange(start, end uint64) (Withdraw
 	withdrawalsByBlockhash := make(map[common.Hash][]db.Withdrawal)
 
 	iter, err := FilterWithdrawalInitiatedWithRetry(s.filterer, &bind.FilterOpts{
-		Start:   start,
-		End:     &end,
-		Context: s.ctx,
+		Start: start,
+		End:   &end,
```

Comment:
Why don't we want to pass these `Context`s in?

Reply:
This context was not being used at all. We're creating a new context for every request and setting it as `opts.Context` = ctxt in `filter.go`.